src/Http/Controllers/Configuration/SsoController.php

```diff
@@ -34,10 +35,22 @@ class SsoController extends Controller
     /**
      * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
      */
-    public function getConfigurationHome()
+    public function getConfigurationHome(Request $request)
     {
 
-        return view('web::configuration.sso.view');
+        $sso_scopes = collect(setting('sso_scopes', true));
+
+        $selected_profile = $sso_scopes->first(function ($item) {
+            return $item->name == 'default';
```

Comment:
You add a `default` property to your object, why not using it instead the name ?
I think it would be more reliant ;)